src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -372,39 +372,39 @@
             </trans-unit>
             <trans-unit id="96">
                 <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
-                <target>Broj elemenata u ovoj kolekciji bi trebalo da bude deljiv sa {{ compared_value }}.</target>
+                <target>Broj elemenata u ovoj kolekciji bi trebalo da bude višestruk u odnosu na {{ compared_value }}.</target>
             </trans-unit>
             <trans-unit id="97">
                 <source>This value should satisfy at least one of the following constraints:</source>
                 <target>Ova vrednost bi trebalo da zadovoljava namjanje jedno od narednih ograničenja:</target>
             </trans-unit>
             <trans-unit id="98">
                 <source>Each element of this collection should satisfy its own set of constraints.</source>
-                <target>Svaki element ove kolekcije bi trebalo da zadovolji sopstveni skup ograničenja.</target>
+                <target>Svaki element ove kolekcije bi trebalo da zadovolji sopstveni set ograničenja.</target>
```

Comment:
I think `skup` is a well-known translation for `set`. We should keep `skup` here.

Reply:
@ajakov You had great suggestions, thank you :)